### PR TITLE
Behnaz / add binary bot

### PR DESCRIPTION
**Pull request by behnaz-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/other-platforms.js [hunk 1]
```diff
@@ -341,6 +343,18 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                     onClick={onClick}
                     otherLinkProps={{ rel: 'noopener noreferrer' }}
                 />
+                {is_eu_country && (
```
Comment (mehrzad-fs) [inline src/components/custom/other-platforms.js:346]: Do we need to duplicate this code for EU and non_eu accounts?

File: src/components/custom/other-platforms.js [hunk 2]
```diff
@@ -364,6 +378,18 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                     onClick={onClick}
                     to="/dbot/"
                 />
+                {!is_eu_country && (
+                    <NavCard
+                        aria_label="BinaryBot"
+                        icon={() => <img src={BinaryBot} alt="" width="32" height="32" />}
+                        content={
+                            <Localize translate_text="Automated trading for our loyal clients who are not ready yet to get more with DBot." />
```
Comment (mehrzad-fs) [inline src/components/custom/other-platforms.js:386]:
```suggestion
                            <Localize translate_text="Our classic &ldquo;drag-and-drop&rdquo; tool for creating trading bots, featuring pop-up trading charts, for advanced users." />
```

File: src/components/custom/other-platforms.js (hunk 1 above)
Comment (behnaz-fs) [reply src/components/custom/other-platforms.js:346]: thanks for checking, yes for eu should show in first column and in non-eu should show in second column as we have derivx for non-eu

File: src/components/custom/other-platforms.js (hunk 2 above)
Comment (behnaz-fs) [reply src/components/custom/other-platforms.js:386]: done

**Review by mehrzad-fs — DISMISSED** [review]

**Review by alamin-deriv — DISMISSED** [review]

**Review by suthesan-deriv — DISMISSED** [review]

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**